.ci/build-platform.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
   platform: "macOS"
-  vmImage: "macOS-10.13"
+  vmImage: "macOS-10.14"
```

Comment:
The macOS tests weren't running at all, because Azure CI no longer supports 10.13

Reply:
Thank you!